app/controllers/api/base_controller/parameters.rb

```diff
@@ -169,11 +171,13 @@ def sort_params(klass)
       end
 
       def sort_directive(attr, order, options)
-        sort_item = attr
-        sort_item = "LOWER(#{sort_item})" if options.map(&:downcase).include?("ignore_case")
-        sort_item << " ASC"  if order && order.downcase.start_with?("asc")
-        sort_item << " DESC" if order && order.downcase.start_with?("desc")
-        sort_item
+        if options.map(&:downcase).include?("ignore_case")
```

Comment:
@imtayadeway so, afaik (spent a bit of time diving into this) it's not possible to add the LOWER to the hash that's needed to sort on the `virtual_attributes`. Because it was getting a little icky, I decided to account for that by adding in another method for sorting of virtual attributes, but not sure how I feel about that. Thoughts, suggestions?

Maybe @kbrock can provide some insight into case-insensitive sort on virtual attributes? `reorder(LOWER(attr)` does not work, and it requires the format `.reorder({:attr => 'asc'})`. Is there a way to add case insensitive sort to this?